examples/ensemble/plot_adaboost_hastie_10_2.py

```diff
@@ -15,29 +15,41 @@
 .. [1] T. Hastie, R. Tibshirani and J. Friedman, "Elements of Statistical
     Learning Ed. 2", Springer, 2009.
 
-.. [2] J. Zhu, H. Zou, S. Rosset, T. Hastie, "Multi-class AdaBoost", 2009.
+..  [2] J Zhu, H. Zou, S. Rosset, T. Hastie, "Multi-class AdaBoost",
+    Statistics and Its Interface, 2009.
 
 """
 
-# Author: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
-#         Noel Dawe <noel.dawe@gmail.com>
+# %%
+# Preparing the data and baseline models
+# ---------------------------------------------------
+# We start by generating the binary classification dataset
+# used in Hastie et al. 2009, Example 10.2.
+
+# Authors: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
+# Noel Dawe <noel.dawe@gmail.com>
 #
 # License: BSD 3 clause
```

Comment:
This may be ultimately a minor detail but, if I recall correctly, to be consistent this block should be moved above this cell. Something to the effect of:
```diff
-29# Authors: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
-30# Noel Dawe <noel.dawe@gmail.com>
-31#
-32# License: BSD 3 clause

+23# Authors: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
+24# Noel Dawe <noel.dawe@gmail.com>
+25#
+26# License: BSD 3 clause
+27
+28# %%
+29# Preparing the data and baseline models
```
If you did notice many notebooks that were rendered in the same manner as what you have here, then I think the point is fine to disregard.

Reply:
> This may be ultimately a minor detail but, if I recall correctly, to be consistent this block should be moved above this cell. Something to the effect of:
> 
> ```diff
> -29# Authors: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
> -30# Noel Dawe <noel.dawe@gmail.com>
> -31#
> -32# License: BSD 3 clause
> 
> +23# Authors: Peter Prettenhofer <peter.prettenhofer@gmail.com>,
> +24# Noel Dawe <noel.dawe@gmail.com>
> +25#
> +26# License: BSD 3 clause
> +27
> +28# %%
> +29# Preparing the data and baseline models
> ```
> 
> If you did notice many notebooks that were rendered in the same manner as what you have here, then I think the point is fine to disregard.

Followed up on this.

These examples have authors as comments in the **same** cell **above** the first code:

- https://scikit-learn.org/stable/auto_examples/ensemble/plot_stack_predictors.html#sphx-glr-auto-examples-ensemble-plot-stack-predictors-py
- https://scikit-learn.org/stable/auto_examples/ensemble/plot_gradient_boosting_early_stopping.html#sphx-glr-auto-examples-ensemble-plot-gradient-boosting-early-stopping-py
- https://scikit-learn.org/stable/auto_examples/ensemble/plot_feature_transformation.html#sphx-glr-auto-examples-ensemble-plot-feature-transformation-py
- https://scikit-learn.org/stable/auto_examples/ensemble/plot_gradient_boosting_regression.html#sphx-glr-auto-examples-ensemble-plot-gradient-boosting-regression-py

At least in the notebook-style examples in `/ensemble/` I have never seen the author cell being completely separate. I think we we should not diverge from that style-choice for the sake of consistency. See my current commit on this. What do you think?